src/components/QuranReader/EndOfScrollingControls/ChapterControls.tsx

```diff
@@ -24,50 +16,15 @@ interface Props {
 }
 
 const ChapterControls: React.FC<Props> = ({ initialData }) => {
-  const isReadingByRevelationOrder = useSelector(selectIsReadingByRevelationOrder);
-
-  const { t, lang } = useTranslation('quran-reader');
-  const scrollToTop = useScrollToTop();
+  const { lang } = useTranslation('quran-reader');
   const chapterIdAndLastVerse = initialData.pagesLookup.lookupRange.to;
   // example : "2:253" -> chapter 2 verse 253
   const chapterId = chapterIdAndLastVerse.split(':')[0];
   const chapterNumber = Number(chapterId);
 
   return (
     <>
-      {!isFirstSurah(chapterNumber, isReadingByRevelationOrder) && (
-        <Button
-          type={ButtonType.Secondary}
-          prefix={<ChevronLeftIcon />}
-          href={getPreviousSurahNavigationUrl(chapterNumber, isReadingByRevelationOrder)}
-          onClick={() => {
-            logButtonClick('chapter_control_prev_chapter');
-          }}
-        >
-          {t('prev-surah')}
-        </Button>
-      )}
-      <Button
-        type={ButtonType.Secondary}
-        onClick={() => {
-          logButtonClick('chapter_control_scroll_to_beginning');
-          scrollToTop();
-        }}
-      >
-        {t('surah-beginning')}
-      </Button>
-      {!isLastSurah(chapterNumber, isReadingByRevelationOrder) && (
-        <Button
-          type={ButtonType.Secondary}
-          suffix={<ChevronRightIcon />}
-          href={getNextSurahNavigationUrl(chapterNumber, isReadingByRevelationOrder)}
-          onClick={() => {
-            logButtonClick('chapter_control_next_chapter');
-          }}
-        >
-          {t('next-surah')}
-        </Button>
-      )}
+      <EndOfSurahSection chapterNumber={chapterNumber} />
```

Comment:
There used to be another order based on the revelation order when that mode is on.